conf/messages.cy

```diff
@@ -341,7 +341,7 @@ subscription.check-your-email.heading=Gwirio’ch cyfeiriad e-bost
 subscription.check-your-email.label=Cyfeiriad e-bost
 subscription.check-your-email.confirm.label=Byddwn yn anfon cysylltiad atoch i gadarnhau’ch cyfeiriad e-bost.
 subscription.check-your-email.confirm.question=Ai dyma’r cyfeiriad e-bost rydych am ei ddefnyddio?
-subscription.check-your-email.yes=Iawn, dyma’r cyfeiriad e-bost rwyf am ei ddefnyddio
+subscription.check-your-email.yes=Iawn
```

Comment:
Normally I'd say don't edit the .en and .cy files - add new keys to -ecc.en and -ecc.cy.  But I think this is Ok